src/data/bucket/symbol_attributes.js

```diff
@@ -6,12 +6,16 @@ import type {StructArrayLayout} from '../../util/struct_array.js';
 export const symbolLayoutAttributes: StructArrayLayout = createLayout([
     {name: 'a_pos_offset',   components: 4, type: 'Int16'},
     {name: 'a_tex_size',     components: 4, type: 'Uint16'},
-    {name: 'a_pixeloffset',  components: 4, type: 'Int16'},
-    {name: 'a_z_tile_anchor', components: 4, type: 'Int16'}
+    {name: 'a_pixeloffset',  components: 4, type: 'Int16'}
+], 4);
+
+export const symbolGlobeExtAttributes: StructArrayLayout = createLayout([
+    {name: 'a_globe_anchor', components: 3, type: 'Int16'},
+    {name: 'a_globe_normal', components: 3, type: 'Float32'},
 ], 4);
 
 export const dynamicLayoutAttributes: StructArrayLayout = createLayout([
-    {name: 'a_projected_pos', components: 3, type: 'Float32'}
+    {name: 'a_projected_pos', components: 4, type: 'Float32'}
```

Comment:
a_projected_pos.z seems to be used only under PROJECTION_GLOBE_VIEW. If so, would it make sense to have it in symbolGlobeExtAttributes, used only for globe view?

Reply:
There is some synergy in keeping all related components in a single vector. For example labels with variable anchors have to have all of the 3 components updated at once. Moving z-component to another buffer would result in wasted bandwidth as we need to update 2 vertex buffer instead of just 1.

Reply:
Clarified offline, buffer update needed on every frame... Thanks @mpulkki-mapbox